docs/CONTRIBUTING.md

```diff
@@ -56,7 +56,7 @@ necessary dependencies get added is to run the following commands:
 ```sh
 make electron-develop
 npm prune --production
```

Comment:
I"m removing this whole section, which described an approach to avoid development dependencies from being shrinkwrapped, which we are doing now.